include/logger/loggerConfig.h

```diff
@@ -260,9 +260,9 @@ typedef struct _PWMConfig{
 } PWMConfig;
 
 /// PWM frequency in Hz.
-#define MAX_PWM_CLOCK_FREQUENCY             200000
-#define MIN_PWM_CLOCK_FREQUENCY	 	10
-#define DEFAULT_PWM_CLOCK_FREQUENCY			10000
+#define MAX_PWM_CLOCK_FREQUENCY             65000
```

Comment:
I don't see it mentioned explicitly either... but I remember that being the standard.

@Jeff-Ciesielski,
   I don't follow your logic.  Perhaps let me rephrase my point:

If the person changing the code is _touching_ the lines anyways (as @brentpicasso was doing in this case), then any lines he touches should be brought up to code standard.  I'm not implying doing that for the whole block per the reason you just mentioned.  Just the lines that are touched.